src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts

```diff
@@ -37,19 +38,25 @@ function buildCurrencyDescriptor(
   amtA: string,
   currencyB: Currency | undefined,
   amtB: string,
-  delimiter = t`for`
+  delimiter = t`for`,
+  locale?: SupportedLocale
 ) {
-  const formattedA = currencyA ? formatCurrencyAmount(CurrencyAmount.fromRawAmount(currencyA, amtA)) : t`Unknown`
+  const formattedA = currencyA
+    ? formatCurrencyAmount({ amount: CurrencyAmount.fromRawAmount(currencyA, amtA), locale })
+    : t`Unknown`
   const symbolA = currencyA?.symbol ?? ''
-  const formattedB = currencyB ? formatCurrencyAmount(CurrencyAmount.fromRawAmount(currencyB, amtB)) : t`Unknown`
+  const formattedB = currencyB
+    ? formatCurrencyAmount({ amount: CurrencyAmount.fromRawAmount(currencyB, amtB), locale })
+    : t`Unknown`
```

Comment:
this only seems to format new history that occurred with the flag on and not any past history. Is this due to caching? You can see this in the screenshot with the recent swap being comma separated and the old history still using a decimal
![Screenshot 2023-09-06 at 9 33 36 AM](https://github.com/Uniswap/interface/assets/11512321/c99640a9-ac90-4bce-9c22-1fd7ec56593e)

Reply:
a few of these use a different formatter function that I haven't gotten to converting yet